File: Daily Test/Java/Character Palindrome Check 31.3.2026.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();

        int[] freq = new int[256];

        for(char c : s.toCharArray())
            freq[c]++;

        int odd = 0;
        for(int i = 0; i < 256; i++)
            if(freq[i] % 2 != 0)
                odd++;

        if(odd <= 1)
            System.out.println("yes");
        else
            System.out.println("no");
    }
}

File: Daily Test/C++/Character Palindrome Check 31.3.2026.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    string s;
    cin >> s;

    unordered_map<char,int> freq;
    for(char c : s)
        freq[c]++;

    int odd = 0;
    for(auto x : freq)
        if(x.second % 2 != 0)
            odd++;

    cout << (odd <= 1 ? "yes" : "no");
}